lib/src/constants/string_data.dart

```diff
@@ -0,0 +1,10 @@
+class StringData {
+  static const String listCoinsTitle = 'Coins';
+
+  static const String emptyList = 'Coin list is empty';
+  static const String hintTextSearch = 'Search';
+  static const String constImage =
+      'https://www.facebook.com/photo/?fbid=2844772675847282&set=a.1403234546667776';
```

Comment:
![image](https://user-images.githubusercontent.com/86453578/142432186-7b30a35c-f94e-498b-85b5-2d179370c06c.png)
lovely girl :v but why she be here?

Reply:
i fixed